spec/controllers/collection_items_controller_spec.rb

```diff
@@ -321,6 +321,43 @@
         end
       end
     end
+
+    context "as an archivist" do
+      let(:archivist) { create(:archivist) }
+      let(:work) { create(:work) }
+
+      let(:collection) do
+        participant = create(:collection_participant, pseud: archivist.default_pseud)
```

Comment:
I was like, wait, _what_, but it turns out the default role in the `collection_participant` factory is owner. 🤷 

https://github.com/otwcode/otwarchive/blob/f7cd4fc80c1482e12d216422759b3eee670b45fb/factories/collections.rb#L12-L15

That seems not particularly ideal (imo, we should rename that factory `collection_owner`), but it's out of scope for this. Maybe we could call the variable `owner` here, though, for a little clarity?